### ISSUE TITLE
Cannot deserialize JSON array on API call (response 400)

### ISSUE THREAD

**Issue by patrikwork1** [reapit/foundations#5277]
**Describe the bug**
I am trying to create a journal entry but on API call the response is - "Cannot deserialize the current JSON array (e.g. [1,2,3]) into type ‘Reapit.Services.JournalEntries.Application.Models.Create.CreateJournalEntryModel' because the type requires a JSON object (e.g. {"name":"value"}) to deserialize To fix this error either change the JSON to a JSON object (e.g. {"name":"value"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array. Path '', line 1, position 1.”. I am sending a JSON object in the body so I am not sure why I am getting this response. 

**To Reproduce**
Steps to reproduce the behaviour eg:
1. Create body as follows - {
  "propertyId": "OXF190022",
  "associatedType": "applicant",
  "associatedId": "OXF190001",
  "description": "John Doe description test text",
  "negotiatorId": "JND"
}
2. POST call to  https://platform.reapit.cloud/journalEntries/ with body above
3. Get error 400 with the above text

**Expected behaviour**
A successful API response 

**Device (please complete the following information for web issues):**
 - Device: [Mac]
 - Browser [Chrome]

**Additional context**
I have used this endpoint in our earlier integration and it worked fine, I am not sure if something have been changed as the API doc looks the same for the journal entry creation endpoint.

**Comment by plittlewood-rpt** [comment]
@patrikwork1 I'm struggling to find this - can you send me the headers you get back in the response (one of them is a request id that should let me locate it)

**Comment by patrikwork1** [comment]
{
    "errors": {
        "": [
            "Cannot deserialize the current JSON array (e.g. [1,2,3]) into type 'Reapit.Services.JournalEntries.Application.Models.Create.CreateJournalEntryModel' because the type requires a JSON object (e.g. {\"name\":\"value\"}) to deserialize correctly.\nTo fix this error either change the JSON to a JSON object (e.g. {\"name\":\"value\"}) or change the deserialized type to an array or a type that implements a collection interface (e.g. ICollection, IList) like List<T> that can be deserialized from a JSON array. JsonArrayAttribute can also be added to the type to force it to deserialize from a JSON array.\nPath '', line 1, position 1."
        ]
    },
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "One or more validation errors occurred.",
    "status": 400,
    "traceId": "|667c5d9-41087fa7163447db."
}

Thats the whole response, I assume you want to see the traceId?

**Comment by patrikwork1** [comment]
Yeah I am positive its not in an array, thats why im a bit confused about the response as supposedly im sending it in the correct format.

**Comment by plittlewood-rpt** [comment]
I've found the request log. It looks like your app isn't serializing the request payload properly as it's coming through us as "[object Object]" rather than the actual JSON string. Can you have a look at your serialisation routines and report back?
